components/dashboard/src/settings/Integrations.tsx

```diff
@@ -199,77 +179,435 @@ function GitProviders() {
+            // GitLab
+            case "read_user": return "Read-only access to your email addresses";
+            case "api": return "Allow making API calls (used to set up a webhook when enabling prebuilds for a repository)";
+            case "read_repository": return "Read/write access to your repositories";
+            // Bitbucket
+            case "account": return "Read-only access to your account information";
+            case "repository": return "Read-only access to your repositories (note: Bitbucket doesn't support revoking scopes)";
+            case "repository:write": return "Read/write access to your repositories (note: Bitbucket doesn't support revoking scopes)";
+            case "pullrequest": return "Read access to pull requests and ability to collaborate via comments, tasks, and approvals (note: Bitbucket doesn't support revoking scopes)";
+            case "pullrequest:write": return "Allow creating, merging and declining pull requests (note: Bitbucket doesn't support revoking scopes)";
+            case "webhook": return "Allow installing webhooks (used when enabling prebuilds for a repository, note: Bitbucket doesn't support revoking scopes)";
+            default: return "";
+        }
+    }
+
     return (<div>
-        <Modal visible={!!diconnectModal} onClose={() => setDisconnectModal(undefined)}>
-            <h3>You are about to disconnect {diconnectModal?.provider.host}</h3>
-            <div>
-                <button onClick={() => disconnect(diconnectModal?.provider!)}>Proceed</button>
-            </div>
-        </Modal>
 
-        <Modal visible={!!editModal} onClose={() => setEditModal(undefined)}>
-            <div>
-                {editModal && (<React.Fragment>
-                    <h3>Permissions granted</h3>
-                    <div>
-                        {editModal && editModal.provider.scopes!.map(scope => (
-                            <div key={`scope-${scope}`}>
-                                <CheckBox
-                                    name={scope}
-                                    desc={scope}
-                                    title={scope}
-                                    key={`scope-checkbox-${scope}`}
-                                    checked={editModal.nextScopes.has(scope)}
-                                    disabled={editModal.provider.requirements?.default.includes(scope)}
-                                    onChange={onChangeScopeHandler}
-                                ></CheckBox>
-                            </div>
-                        ))}
+        {diconnectModal && (
+            <Modal visible={true} onClose={() => setDisconnectModal(undefined)}>
+                <h3 className="pb-2">Disconnect Provider</h3>
+                <div className="border-t border-b border-gray-200 mt-2 -mx-6 px-6 py-4">
+                    <p className="pb-4 text-gray-500 text-base">Are you sure you want to disconnect from the following provider?</p>
```

Comment:
**suggestion**: Minor rephrasing to match the voice and tone with the rest of the product.

```suggestion
                    <p className="pb-4 text-gray-500 text-base">Are you sure you want to disconnect the following provider?</p>
```